ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py

```diff
@@ -112,6 +113,12 @@ class DashboardServiceTopology(ServiceTopology):
                 processor="yield_dashboard",
                 consumer=["dashboard_service"],
             ),
+            NodeStage(
+                type_=User,
+                context="owner",
+                processor="yield_owner",
```

Comment:
We are not exactly yielding anything here. Maybe change the method name to `process_owner`